docker/linux-headless/browsers.ini

```diff
@@ -0,0 +1,3 @@
+[Chrome]
+exe=/usr/bin/google-chrome

```

Comment:
I think this might need to be ```exe=/opt/google/chrome/chrome```.  Otherwise terminating will try to kill google-chrome processes instead of chrome.